p/pear_archive-tar/pear_archive-tar_1.4.11_1.4.12_rhel_8.4.sh

```diff
@@ -0,0 +1,59 @@
+! /bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package       : pear/archive_tar
+# Version       : 1.4.11
+# Source repo   : https://github.com/pear/Archive_Tar
+# Tested on     : RHEL 8.4
+# Maintainer    : Amit Baheti <aramswar@in.ibm.com>
+#
+# Pre-requisite : pear package manager should be available on the system
+#                 as it needs user interaction for installation and cannot 
+#                 cannot be installed using script. Install as specified on
+#                 https://pear.php.net/manual/en/installation.getting.php
```

Comment:
I am able to install pear using "yum install php-pear -y" on RHEL and run the tests successfully. Please check if this is possible.

Reply:
Done